doc/UserGuide.md

```diff
@@ -887,6 +887,46 @@ class will effectively use 1 for all \kappa's.
 By default, a 3x3 or 3x3x3 neigbourhood is used where the weights are set
 to x-voxel_size divided by the Euclidean distance between the points.
 
+##### LogcoshPrior (PET)
+
+This implements a Logcosh. The gradient of the prior is computed as follows:
+
+    g_r = \sum_dr w_{dr} \frac{1}{s} \tanh (s (\lambda_r-\lambda_{dr}))* \kappa_r * \kappa_{r+dr}
```

Comment:
When using LaTex in a markdown file, need some $$ I guess. See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions. Or avoid the latex

Reply:
Also, it makes more sense to give the prior than the gradient. From the STIR doc with a correction
```
f = \sum_{r,dr} w_{dr} \frac{1}{2 scalar^2}  log(cosh(scalar(\lambda_r - \lambda_{r+dr})))  \kappa_r  \kappa_{r+dr}
```
$$
f = \sum_{r,dr} w_{dr} \frac{1}{2 scalar^2}  log(cosh(scalar(\lambda_r - \lambda_{r+dr})))  \kappa_r  \kappa_{r+dr}
$$

Reply:
@KrisThielemans I tried to follow the convention for quadratic prior where only the gradient is given and was given in a LaTeX formula without using $$s. That said, the value of that quadratic prior is very straightforward as compared to Logcosh/relative difference prior. I did take the definition of the gradient for STIR userguide: https://stir.sourceforge.net/documentation/STIR-UsersGuide.pdf
![image](https://github.com/SyneRBI/SIRF/assets/72553490/7a1ea7f7-2465-486c-81e8-3e40eb2eae3f)